examples/js-console/src/main/webapp/keycloak.js

```diff
@@ -0,0 +1 @@
+/home/st/dev/keycloak/adapters/oidc/js/src/main/resources/keycloak.js
```

Comment:
guess this was changed by accident?

Reply:
Yer, wasn't meant to commit that file. Removed